ultimatejobweb/templates/available_jobs.html

```diff
@@ -61,19 +66,28 @@ <h1 class="display-3">UltimateJob</h1>
         </div>
 
    <!--tables-->
-   <div class="container">
+	<div class="container">
     <table class="table table-hover table-stripped">
         <thead class="thead-dark">
             <tr>
                 <th>Company name</th>
                 <th>Position offered</th>
-                <th>Upload date</th>
-                <th>Location</th>
                 <th>Link to job application</th>
             </tr>
         </thead>
         <tbody>
-            <tr>
+		   <h1> Available jobs  </h1>
+		   <div>
+				{% for item in job_posts %}
+				<tr> 
+					<td>{{ item.company }}</td>
+					<td>{{ item.job_title }}</td>
+					<td>{{ item.description_url }}</td>
```

Comment:
My recommendation for this would be:
<td><a href="{{ item.description_url }}">{{ item.job_title }}</a></td>
so that the title is a clickable URL.